### DISCUSSION TITLE
Is it possible to effectively implement equalizer for music in osu?

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#20554] [Ideas] [answered]
In my music player I can tune the music as I like. 
![изображение](https://user-images.githubusercontent.com/55319814/193502450-0c5678fa-5185-49ec-bdf2-5849381a2ec3.png)

It's much better than default settings. Especially for osu songs... They are boosted for maximize volume. Almost every of them. 
It would be better to implement ingame equalizer that "remake" song file to adjusted temporary one, before playing (to avoid lags). In osu menu it can be realtime processing. 
I often get annoyed by the way the music is set up and it ruins the fun of the game. For some people this is really important.

**Comment by ghost** [comment]
Strongly dislike this idea. osu! is not a music player, so I feel that issues like these are just out of scope

**Reply by okolenmi** [reply]
It's an option. Just don't turn on this. It will be very useful for players like me. Why try to remove something that will not affect you in any way? Or will it affect? Maybe some specific cheats that depends on song? -_-

**Comment by peppy** [comment, marked as answer]
This is out of scope. Please install a system-level equaliser.

**Reply by okolenmi** [reply]
Latency. Big latency. It's not a solution.
